protobuf_deps.bzl

```diff
@@ -110,8 +110,8 @@ def protobuf_deps():
         _github_archive(
             name = "rules_ruby",
             repo = "https://github.com/protocolbuffers/rules_ruby",
-            commit = "8fca842a3006c3d637114aba4f6bf9695bb3a432",
-            sha256 = "2619f9a23cee6f6a198d9ef284b6f6cbc901545ee9a9aac9ffa6b83dbf17cf0c",
+            commit = "b7f3e9756f3c45527be27bc38840d5a1ba690436",
```

Comment:
https://github.com/protocolbuffers/rules_ruby/commit/b7f3e9756f3c45527be27bc38840d5a1ba690436 shows 
> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

It seems this commit is not part of the `main` branch is has been deleted